mqtt/Dockerfile.armv7

```diff
@@ -0,0 +1,30 @@
+# -*- mode: dockerfile -*-
```

Comment:
This docker file looks very similar to one we have already. And it supports `--build-arg BASE_IMAGE=...`